### DISCUSSION TITLE
Translate the install to anothers languages

### DISCUSSION THREAD

**Discussion by erikadeolima** [chris1111/Wireless-USB-Big-Sur-Adapter#192] [Ideas]
Hi, sorry, i'm new at github, and i don't known how to send this idea.... I'm brazilian, and although speak a little English, I found it difficult to use the install instructions. I believe that it would be greatly improved by replacing the images with step-by-step texts, as the images are of low quality and this made me confused, or even the translation, to Spanish or Portuguese, but I know that would be much more work.
for example, at the end of the installation, when we have to authorize the usb device, the image is self-explanatory, but it is bad to visualize, if it were replaced by a
Go to system preferences>network>..... (sorry I don't remember the rest of the image), it would be much better.

**Comment by chris1111** [comment]
I @erikadeolima Thanks for your comment
I can not translate to other language, if you whant translating the file, you can do it by your self and send me the translation files
I will add your language with pleasure
Files is attaching here
[en.lproj.zip](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/files/8156622/en.lproj.zip)

**Reply by erikadeolima** [reply]
Hi @chris1111 thanks for the answer! Yes, I translate, with great pleasure... here are the files.

[pt.lproj.zip](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/files/8162790/pt.lproj.zip)

**Reply by erikadeolima** [reply]
And I added the updates in the conclusion, i think this way is better to understand
[en.lproj 2.zip](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/files/8162807/en.lproj.2.zip)
.

**Comment by chris1111** [comment]
Thank you verry much 
[Release done](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/releases/tag/V12)
